src/examples/unofficial/go.md

```diff
@@ -0,0 +1,112 @@
+<!-- markdownlint-disable MD014 -->
+
+# [![CS](https://cdn.discordapp.com/emojis/1128611804596158574.webp?size=24&quality=lossless)](https://nekos.best/discord?ref=docs) Go
+
+---
+
+## Installation
+Add the module to your project
+```bash
+$ go get -u github.com/Yakiyo/nekos_best.go
+```
+
+## Usage
+Import it in your code
+```go
+import (
+    nb github.com/Yakiyo/nekos_best.go
+)
+```
+Full reference at [go.pkg.dev](https://pkg.go.dev/github.com/Yakiyo/nekos_best.go)
+
+### Fetch a single entry
+```go
+res, err := nb.Fetch("neko")
+if err != nil {
+    // handle err
+}
+fmt.Println(res.Url, res.Artist_name, res.Artist_href)
+```
+
+### Fetch multiple entries
+```go
+res, _ := nb.FetchMany("baka", 3)
+
+fmt.Println(res[0].Url)
+```
+
+### Fetch a file
+```go
+import "os"
+
+res, _ := nb.FetchFile("pat")
+
+os.WriteFile("image.png", res.Data, 0644)

```

Comment:
The API no longer supports numbered IDs, please change your example to use a UUID.